src/main/java/ca/on/oicr/gsi/dimsum/service/filtering/CaseFilterKey.java

```diff
@@ -47,7 +47,34 @@ public Function<String, Predicate<TestTableView>> testTableViewPredicate() {
   PROJECT(string -> kase -> kase.getProjects().stream()
       .anyMatch(project -> project.getName().equalsIgnoreCase(string))),
   REQUISITION(string -> kase -> kase.getRequisition().getName().toLowerCase().startsWith(string.toLowerCase())),
-  REQUISITION_ID(string -> kase -> kase.getRequisition().getId() == Long.parseLong(string));
+  REQUISITION_ID(string -> kase -> kase.getRequisition().getId() == Long.parseLong(string)),
+  TEST(string -> kase -> kase.getTests().stream().anyMatch(test -> test.getName().equalsIgnoreCase(string))),
```

Comment:
Just to confirm, is this to filter the tests with specific name in `Test` page?

Reply:
Yeah. For example, say we have a case with 2 tests - "Tumour WG" and "Normal WG" and filter by "Test: Tumour WG"
* If we are looking at the Cases table, we should see this case because one of its tests matches
* If we are looking at the Tests table, we should only see the "Tumour WG" test
  * currently, the "Normal WG" test would also be included 
* If we are looking at the Extractions table, we should only see the extractions for the "Tumour WG" test
  * currently, the extractions for the "Normal WG" test would also be included